bodhi/server/static/js/update_form.js

```diff
@@ -26,314 +26,192 @@ $(document).ready(function() {
-            var bug = {id: parseInt($(this).val()), title: $(this).parent().text().replace(/^#\d+\s/m, '')};
-            checked_bugs.push(bug);
-        });
-        var checked_candidates = [];
-        $("#candidate-checkboxes input:checkbox:checked").each(function(){
-            var buildid = $(this).attr('data-build-id')
-            if (buildid !== '') {
-                checked_candidates.push([$(this).val(), parseInt(buildid)]);
-            } else {
-                checked_candidates.push([$(this).val(), false]);
-            }
-        });
-        // Empty lists
-        document.getElementById("candidate-checkboxes").innerHTML = "<img class='spinner' src='static/img/spinner.gif'>";
-        document.getElementById("bugs-checkboxes").innerHTML = "<img class='spinner' src='static/img/spinner.gif'>";
-        // Get the candidate builds
-        $.ajax({
-            url: 'latest_candidates',
-            timeout: 10000,
-            data: $.param({package: datum.name}),
-            success: function(builds) {
-                if (builds.length == 0) {
-                    return messenger.post({
-                        message: 'No candidate builds found for ' + datum.name,
-                        type: 'info',
-                    });
-                }
-                $.each(builds, function(i, build) {
-                    // Insert the checkbox only if this ID is not already listed
-                    if ($.inArray(build.id, checked_candidates) == -1) {
-                        add_build_checkbox(build.nvr, build.id, false, false);
+    var $bugs_search_selectize = $('#bugs-search').selectize( {
+        create: function(input, callback){
+            $("#bugs-card .selectize-control").addClass("loading")
+            $.ajax({
+                url: 'https://bugzilla.redhat.com/rest/bug?id=' + encodeURIComponent(input),
+                type: 'GET',
+                dataType: 'jsonp',
+                error: function() {
+                    callback();
+                },
+                success: function(data) {
+                    if (data.bugs.length != 1) {
+                        messenger.post({
+                            message: 'Cannot find data for bug #' + input + '. Either the bug is private or doesn\'t exist.',
+                            type: 'error',
+                        });
+                        $("#bugs-card .selectize-control").removeClass("loading")
+                        callback();
+                    } else {
+                        // Check Bug product
+                        var product = data.bugs[0].product;
+                        if (settings.bz_products.indexOf(product) == -1) {
+                            messenger.post({
+                                message: 'Bug #' + data.bugs[0].id + ' doesn\'t seem to refer to Fedora or Fedora EPEL.\nAre you sure you want to reference it in this update? Bodhi will not be able to operate on this bug!',
```

Comment:
Should we remove the reference to Fedora and be more generic?

Reply:
Okay, updated this string to read:

    doesn\'t seem to refer to a product that Bodhi manages updates for.\nAre you sure you want to reference it in this update? Bodhi will not be able to operate on this bug!